lib/migration-chunks/slicers/into-separate-chunk.js

```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+const createPredicateSlicer = require('./predicate');
+
+const returnTrueOnce = function () {
```

Comment:
nit: this seems a bit restrictive to be used by other slicers in the future and it also has implicit knowledge of how the `field-id-change` and `field-delete` chunks are formed.

Reply:
Not sure I understand, this slicer will put an item of a given type into its own chunk and will work for any type of item. Can you explain where the implicit knowledge is?

Reply:
By implicit knowledge I meant the fact that `field-id-change` and `field-delete` expect to be called only once since field deletion and id change have to always go on independent chunks. But this not explicitly stated anywhere. Also, if we later add another slicer which needs to group by more than one step this won't work.

Reply:
I see... To me, the module name `into-separate-chunk` along with the way it's called
```
const sliceByFieldDelete = require('./slicers/into-separate-chunk')('field/delete');
```
is explicit enough. I could add a comment for `returnTrueOnce` that explains that?
As for other slicer that need to group differently: Sure, the other slicers remain the same, and we can always add different ones as well. But if we encounter another case where a step of a type needs to be alone in a chunk, we can use this one.